source/base/polynomials_bernstein.cc

```diff
@@ -38,11 +17,13 @@ namespace Binomial
     AssertIndexRange(k, n+1);
     std::vector<number> coeff(n + 1, number(0.0));
     for (unsigned int i = k; i < n + 1; ++i)
-      coeff[i] = (pow(number(-1), number(i - k)) * binomial(n, i)
-                  * binomial(i, k));
+      {
+        coeff[i] = ((i - k) % 2 == 0 ? 1 : -1)
+                   * boost::math::binomial_coefficient<number>(n, i)
+                   * boost::math::binomial_coefficient<number>(i, k);
```

Comment:
> We do have this header file in bundled/ indeed.

Let me rephrase. My system has 18 libboost_math_* support libraries
installed. But, yes, you're right. From their contents (I just checked
the boost sources) they're very likely not needed for our use case.